elro/device.py

```diff
@@ -91,6 +92,19 @@ def battery_level(self, battery_level):
         self._battery_level = battery_level
         self._send_update_event()
 
+    @property
+    def signal_strength(self):
+        """
+        The current signal strength of the device from 0 to 4.
```

Comment:
It would be best to add this comment to the readme so people don't need to browse the code to know the signal scale.